src/EditorFeatures/Core.Wpf/InlineRename/UI/Adornment/RenameFlyout.xaml.cs

```diff
@@ -200,5 +201,15 @@ private void ToggleExpand(object sender, RoutedEventArgs e)
         {
             _viewModel.IsExpanded = !_viewModel.IsExpanded;
         }
+
+        private void IdentifierTextBox_SelectionChanged(object sender, RoutedEventArgs e)
+        {
+            var start = IdentifierTextBox.SelectionStart;
+            var length = IdentifierTextBox.SelectionLength;
+
+            var textStartPoint = _viewModel.InitialTrackingSpan.GetStartPoint(_textView.TextSnapshot);
```

Comment:
small concert about using TextSnapshot here.  does this work for projection buffers?  do you have the actual subject buffer we're applying to?

(note: this may be irrelevant if everything here expects to work at that layer.  but it's a bit suspect compared to normal roslyn patterns for view/buffers/snapshots).

Reply:
Verified, and changed to use the same buffer the `InitialTrackingSpan` was created with. That will make sure we're doing the right thing with projection buffers